### DISCUSSION TITLE
Smith crashing canvas

### DISCUSSION THREAD

**Discussion by felix-Ggasex** [thatskymod/Sky-CotL-Scripts#45] [General]
I just updated sky and every time I try to play through canvas, smith completely crashes my game. Also aw4c isn't working at all

**Comment by thatskymod** [comment]
- **libsmite:** It appears that the recent Sky update has indeed rendered libsmite ineffective due to substantial changes introduced in version 0.22.0. As a result, it is being removed from the repository until a solution is identified. Thanks for bringing this to attention.
- **libaw4c:** As an alternative to AW4C, consider using libdukun.so and please report back if it works. Please note that the traditional AW4C has reached its end-of-life and will no longer receive official updates or support.

**Comment by thatskymod** [comment]
# Update July 19, 2023:

It is recommended to use [Star](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Star_v0.1.0.apk) ModLoader instead of Canvas, as Canvas is deprecated and no longer maintained. There is also an updated version of Smite with several improvements available for download. Please note that [libsmite-1.0.3a.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libsmite-1.0.3a.so) and subsequent versions **are only compatible with Star. You can no longer install Smite in traditional Canvas.**

Due to inactivity and nature of this issue, this ticket will be closed, as it appears to be no longer relevant. Please feel free to re-open should you require further assistance.

**Comment by RandelsRedTeaPot** [comment]
Please remove your MOD FILE from MOD Manager , Remove All LIB File , Try to add 1 by 1 to see which LIB was corrupted/not working , my Canvas App still the old one and it run normally without any LIB File , DONT USE LIB LANGIT OR LIB SMITE! BOTH NOW ARE CORRUPTED/NOT WORKING AND IT WILL KICK YOU FROM ACCESSING CANVAS!

and these 2 LIB FILE was the REASON why you got kicked from canvas , don't worry , your canvas still working fine , JUST THE LIB FILE isn't , REJECTED FROM ACCESSING CANVAS/SKY
